services/src/main/java/io/grpc/services/BinlogHelper.java

```diff
@@ -119,14 +119,17 @@ void logSendInitialMetadata(
         CallId callId) {
       Preconditions.checkArgument(methodName == null || !isServer);
       Preconditions.checkArgument(timeout == null || !isServer);
+      // In java, we strip the leading '/'. To be consistent with the rest of gRPC we must
```

Comment:
Java does not "strip" the leading slash. It just doesn't include it.

Reply:
I'll change the wording to be more clear. I was thinking of this code path https://github.com/grpc/grpc-java/blob/master/netty/src/main/java/io/grpc/netty/NettyServerHandler.java#L392